spec/adapters/alma_adapter/availability_status_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe AlmaAdapter::AvailabilityStatus do
+  describe "#bib_availability_from_items" do
+    before do
+      stub_alma_holding_items(mms_id: "99125379706706421", holding_id: "ALL", filename: "99125379706706421_holding_items.json", query: "order_by=enum_a")
+      stub_alma_library(library_code: "firestone", location_code: "res3hr")
+      stub_alma_library(library_code: "firestone", location_code: "stacks")
+      stub_alma_library(library_code: "firestone", location_code: "dixn")
+      stub_alma_holding_items(mms_id: "99125378873406421", holding_id: "ALL", filename: "99125378873406421_holding_items.json", query: "order_by=enum_a")
+      stub_alma_library(library_code: "marquand", location_code: "res")
+    end
+
+    it "reports available when all items are available" do
+      bib = Alma::Bib.new("mms_id" => "99125379706706421")
```

Comment:
did you want to use a different json for all items be available? the one '99125379706706421_holding_items.json' which is stubbed has a variety of items statuses and you are using it in the third test

Reply:
No, I don't.  

The fixture in `99125379706706421` has two holdings that represent two different conditions (all available in one case, some available in the other).

Reply:
Thank you. I missed the 2 holdings.